<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickConnectWindow</class>
 <widget class="QDialog" name="QuickConnectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelProtocol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProtocol">
         <item>
          <property name="text">
           <string notr="true">Telnet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Serial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Local Shell</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Raw</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Named Pipe</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">SSH2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">VNC</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelHostname">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hostname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHostname"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxHostname"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRefresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPort"/>
       </item>
       <item>
        <widget class="QLabel" name="labelWebSocket">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>WebSocket</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxWebSocket">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Insecure</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Secure</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxRawMode">
         <item>
          <property name="text">
           <string>TCP Client</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP Server</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UDP Send</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UDP Recv</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelUsername">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsername"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelDataBits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>DataBits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDataBits">
         <item>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelParity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxParity">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelStopBits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>StopBits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxStopBits">
         <item>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFlowCtrl">
         <property name="text">
          <string notr="true">RTS/CTS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxXEnable">
         <property name="text">
          <string notr="true">XON/XOFF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBoxSaveSession">
         <property name="text">
          <string>Save session</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOpenInTab">
         <property name="text">
          <string>Open in tab</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxProtocol</tabstop>
  <tabstop>lineEditHostname</tabstop>
  <tabstop>comboBoxHostname</tabstop>
  <tabstop>pushButtonRefresh</tabstop>
  <tabstop>spinBoxPort</tabstop>
  <tabstop>comboBoxWebSocket</tabstop>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>comboBoxDataBits</tabstop>
  <tabstop>comboBoxParity</tabstop>
  <tabstop>comboBoxStopBits</tabstop>
  <tabstop>checkBoxFlowCtrl</tabstop>
  <tabstop>checkBoxXEnable</tabstop>
  <tabstop>checkBoxSaveSession</tabstop>
  <tabstop>checkBoxOpenInTab</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuickConnectWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickConnectWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
